### ISSUE TITLE
Adding installation instructions and configuration examples

### ISSUE THREAD

**Issue by fuerst** [leighgarbs/sproxy#1]
Thanks for this Bounjour sleep proxy implementation! Runs well at my Arch Linux NAS. Do you mind adding my installation documentation as well as configuration examples to the repository?

**Comment by leighgarbs** [comment]
Yes, I could do that.  Do you have a pull request I can take a look at?
